config-templates/template_list.json

```diff
@@ -94,6 +94,14 @@
     "default-version": "1.4.0",
     "enabled": "true",
     "provider": "ibm"
+  }, 
+  {
+    "name": "ibm-vpc-block-csi-driver",
+    "displayname": "IBM VPC Block CSI driver",
```

Comment:
Pls add [Beta] 
`"IBM VPC Block CSI driver"` => `"[Beta] IBM VPC Block CSI driver"`
and `"Description": "IBM VPC Block CSI driver, Provides block storage on VPC.",` => `"Description": "[Beta] IBM VPC Block CSI driver, Provides block storage on VPC.",`